log4rs-logstash/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "log4rs-logstash"
+version = "0.1.0"
+edition = "2018"
```

Comment:
You will need this keys for release to crates.io
```toml

name = "log4rs-logstash"
version = "..."
description = "..."
repository = "..."
homepage = "..."
documentation = "..."
keywords = ["...", "..."]
categories = ["...", "..."]
license = "MIT"
authors = ["..."]
edition = "2018"
readme = "..."
```

Reply:
Done